components/script/dom/imagedata.rs

```diff
@@ -80,8 +80,9 @@ impl ImageDataMethods for ImageData {
         self.height
     }
 
+    #[allow(unsafe_code)]
     // https://html.spec.whatwg.org/multipage/#dom-imagedata-data
-    fn Data(&self, _: *mut JSContext) -> *mut JSObject {
+    unsafe fn Data(&self, _: *mut JSContext) -> *mut JSObject {
```

Comment:
The context looks to be ignored here (and also in other places). Should we make this constructor unsafe anyway?